game/scripts/vscripts/modules/hero_selection/util.lua

```diff
@@ -113,14 +113,16 @@ function TransformUnitClass(unit, classTable, skipAbilityRemap)
 			unit:SetBaseDamageMax(value)
 		elseif key == "AttackRate" then
 			unit:SetBaseAttackTime(value)
+			unit:SetNetworkableEntityInfo("AttackRate", value)
 		elseif key == "AttackAcquisitionRange" then
 			unit:SetAcquisitionRange(value)
 		elseif key == "ProjectileModel" then
 			unit:SetRangedProjectileName(value)
 		elseif key == "AttributePrimary" then
-			Timers:CreateTimer(0.1, function()
+			Timers:CreateTimer(1, function()
 				unit:SetPrimaryAttribute(_G[value])
-			end)
+				unit:CalculateStatBonus()
+			end, nil, true)
```

Comment:
Won't `Timers:CreateTimer(0, function()` work?
I don't understand a reason to add tick-based timers, isn't `TIMERS_THINK * n` the same?

Reply:
Don't dare, might break because multithreading/floating point/valve

The original alr breaks half the time with illus